packages/docs-site/src/components/clickToCopy.tsx

```diff
@@ -38,8 +36,7 @@ export interface IClickToCopyState {
  *  - `[data-copied-message="<message>"]` will be shown when the element has been copied.
  * The message is reset to default when the user mouses off the element after copying it.
  */
-@PureRender
-export class ClickToCopy extends React.Component<IClickToCopyProps, IClickToCopyState> {
+export class ClickToCopy extends React.PureComponent<IClickToCopyProps, IClickToCopyState> {
```

Comment:
just seemed like making this change now would bloat the number of files affected in this PR unnecessarily. but yeah, no harm in it.